biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -99,6 +101,15 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	public static EE highestFromVersion(String versionString) {
```

Comment:
The _version_ referred to here is unclear. It is the compiler _target version_. So perhaps a name of `highestFromTargetVersion` is a more descriptive name?